docs/software/markdown-migration.md

```diff
@@ -78,8 +78,8 @@ Make sure to use the TWiki document as a reference when making fixes!
 
 Within command blocks and file snippets, we've used `%RED%...%ENDCOLOR%`, `&lt;...&gt;`, etc. to highlight areas that users would have to insert text specific to their site. For now, use desciptive, all-caps text wrapped in angle brackets to indicate user input:
 
-```
-[user@client ~]$ condor_ce_trace -d <CE HOSTNAME>
+```console
+[root@client ~]# condor_ce_trace -d <CE HOSTNAME>
```

Comment:
It doesn't seem to handle the `#` well, it looks like a comment with the grayed out font